frontend/src/components/OtherImplementations.tsx

```diff
@@ -38,22 +41,44 @@ export const OtherImplementations = ({ otherImplementationsData }: Props) => {
           {(props) => (
             <Popover id="popover-basic" {...props}>
               <Popover.Body>
-                {Object.entries(otherImplementationsData).map(([id, impl]) => {
-                  const implementationPath = getImplementationPath(id);
-                  return (
-                    <p key={id}>
-                      <NavLink
-                        style={{ fontSize: "1rem", fontWeight: "bold" }}
-                        to={`/implementations/${implementationPath}`}
-                      >
-                        {impl.name}
-                      </NavLink>
-                      <span className="ps-1 text-body-secondary fw-bold">
-                        {mapLanguage(impl.language)}
-                      </span>
-                    </p>
-                  );
-                })}
+                <Container className="p-0">
+                  <Row className="flex-column">
+                    {otherImplementationsDataArray.map(([id, impl], index) => {
+                      const implementationPath = getImplementationPath(id);
+                      const latestSupportedDialect =
+                        getLatestSupportedDialect(impl);
+                      return (
+                        <Col
+                          key={id}
+                          className={
+                            index === otherImplementationsDataArray.length - 1
+                              ? ""
+                              : "mb-2"
+                          }
+                        >
+                          <div>
+                            <NavLink
+                              style={{ fontSize: "1rem", fontWeight: "bold" }}
+                              to={`/implementations/${implementationPath}`}
+                            >
+                              {impl.name}
+                            </NavLink>
+                            <span className="ps-1 text-body-secondary fw-bold">
+                              {mapLanguage(impl.language)}
+                            </span>
+                          </div>
+                          <span className="text-body-secondary">
+                            (LTS Dialect:{" "}
```

Comment:
1. is LTS really an abbreviation of "latest"?
2. Please dons' use spaces for positioning elements - use CSS classes instead

Reply:
should I change LTS to "Latest Supported Dialect" then ?

Reply:
Not really feeling strong in english writing :) but I'd say yes - and I don't know if capitalization is correct here? It's not some kind of title or anything

Reply:
> actually as it's text next to other text, it's better the way it is

So do I leave it as it is ?

Reply:
> So do I leave it as it is ?

Yep